docs/pipeline.rst

```diff
@@ -143,3 +143,41 @@ DataNode
 --------
 .. autoclass:: nvidia.dali.pipeline.DataNode
    :members:
+
+Pipeline Debug Mode (experimental)
+------------------
+
+Pipeline can be run in debug mode by replacing :meth:`@nvidia.dali.pipeline_def <nvidia.dali.pipeline_def>` decorator with its experimental variant
+``@nvidia.dali.experimental.pipeline_def`` and setting parameter ``debug`` to True. It allows you
```

Comment:
Do we need the separate decorator for it? Wouldn't it be sufficient to just mark the `debug` option as experimental, not the whole decorator?

Reply:
I didn't want to change the regular decorator, that's why we did a separate one, similar to inputs in tf dataset.  In the end we will probably merge them, but I think it's hard to change something in official api if we want to do it.